### PR TITLE
feat/replace-local-mysql-with-aws-rds

### PR DESCRIPTION
**Pull request by cbx6664**

### CODE REVIEW

File: IntelliEdu-application-service/src/main/resources/application-prod.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    store-type: redis
+    timeout: 2592000 # 30 days expiration
+    redis:
+      # 禁用 Spring Session 自动配置 Redis 的行为，确保使用云服务商提供的 Redis 配置。
+      # 这行配置可以防止 Spring Boot 在启动时尝试对 Redis 进行任何配置更改，从而避免潜在的连接问题或配置冲突
+      configure-action: none
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: ${INTELLIEDU_APPLICATION_MYSQL_HOST}
+    username: ${INTELLIEDU_ANSWER_RECORD_MYSQL_USERNAME}
```
Comment (cpuoverload) [inline IntelliEdu-application-service/src/main/resources/application-prod.yml:18]: wrong name?

File: IntelliEdu-application-service/src/main/resources/application-prod.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    map-underscore-to-camel-case: true
+    log-impl: org.apache.ibatis.logging.stdout.StdOutImpl
+  global-config:
+    db-config:
+      logic-delete-field: deleted
+      logic-delete-value: 1
+      logic-not-delete-value: 0
+knife4j:
+  enable: true
+logging:
```
Comment (cpuoverload) [inline IntelliEdu-application-service/src/main/resources/application-prod.yml:43]: delete log in production env

File: IntelliEdu-application-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -8,17 +8,23 @@ spring:
...
+      # 禁用 Spring Session 自动配置 Redis 的行为，确保使用云服务商提供的 Redis 配置。
+      # 这行配置可以防止 Spring Boot 在启动时尝试对 Redis 进行任何配置更改，从而避免潜在的连接问题或配置冲突
+      configure-action: none
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
-    url: jdbc:mysql://localhost:3306/intelli_edu_application
-    username: root
-    password: ${MYSQL_PASSWORD}
+    url: ${INTELLIEDU_APPLICATION_MYSQL_HOST}
+    username: ${INTELLIEDU_ANSWER_RECORD_MYSQL_USERNAME}
```
Comment (cpuoverload) [inline IntelliEdu-application-service/src/main/resources/application.yml:18]: wrong name?

File: IntelliEdu-application-service/src/main/resources/application-prod.yml (hunk 1 above)
Comment (cbx6664) [reply IntelliEdu-application-service/src/main/resources/application-prod.yml:18]: yes

File: IntelliEdu-application-service/src/main/resources/application-prod.yml (hunk 2 above)
Comment (cbx6664) [reply IntelliEdu-application-service/src/main/resources/application-prod.yml:43]: ok

**Pull request merged**